include/osg/ref_ptr

```diff
@@ -141,6 +156,20 @@ class ref_ptr
             if (tmp_ptr) tmp_ptr->unref();
         }
 
+#if __cplusplus >= 201103L
+        template<class Other> void moveAssign(ref_ptr<Other>&& rp)
```

Comment:
This is only used by the move assignment operator, perhaps there is no need for this separate function?

Reply:
Maybe, but I wasn't sure what's the reasoning behind copy-assignment with the similar approach. Even there are 3 different definitions for it and two of them have common implementation, there is no actual difference.

Reply:
Done